pgjdbc/src/main/java/org/postgresql/jdbc/EscapedFunctions.java

```diff
@@ -135,7 +136,7 @@
    * @return a Method object or null if not found
    */
   public static Method getFunction(String functionName) {
-    return functionMap.get("sql" + functionName.toLowerCase(Locale.US));
+    return functionMap.get(functionName.toLowerCase(Locale.US));
```

Comment:
Is `toLowerCase` required here?
I would suggest removing it as well (e.g. cache both variations for the function name)

Reply:
should keep it to compatible with query in the wild eg RounD vs round

Reply:
the map has keys based on the method names in this class, which are all lower case, right?

Reply:
Yes

**before**
key = "sql"+function (eg `sqlweek`) lowercased
lookup done with "sql" + function name lowercased (memory allocation show in profiling)

**After**
key = function (eg `week`) lowercased
lookup done with function name lowercased (memory still allocated by the lowercase call but it's better)